jitbuilder/release/Makefile

```diff
@@ -44,6 +45,7 @@ ALL_TESTS = \
             toiltype
 
 goal: $(ALL_TESTS)
+	./transactionaloperations
```

Comment:
In my service, I handled the situation where the platform doesn't support xbegin by directly going to persistentFailure path. I tested my test case both on platform does support and the one doesn't.